src/tests/src/io/yml-importers/testConvertorVisitor.cpp

```diff
@@ -325,3 +325,38 @@ BOOST_AUTO_TEST_CASE(TimeShiftExpressionMul)
     Visitors::CompareVisitor cmp;
     BOOST_CHECK(cmp.dispatch(expr.node, timeShiftNode));
 }
+
+BOOST_AUTO_TEST_CASE(TimeSumExpression)
```

Comment:
Now it's done. 
But does this have to be this complicated ?

Reply:
Because the concept is not easy